wiki/Options/Keyboard_Bindings/en.md

```diff
@@ -0,0 +1,153 @@
+# Keyboard Bindings
+
+To access this dialog, open the [Options](/wiki/Options/), type in `change` then click on `Change keyboard bindings`.
+
+Here, you can change reassign keyboard keys with the listed actions.
+
+**Be aware** that the osu!client will not notify you for keys used multiple times (this includes conflicting keys)!
+
+## osu! Standard Mode
+
+| Name        | Default |
```

Comment:
/shrug if I did that then I would need to add more spaces to the columns... `Default` seems to be self-explanatory.